recipes/configs/llama3/8B_qlora_single_device.yaml

```diff
@@ -44,7 +44,7 @@ save_adapter_weights_only: False
 # Dataset and Sampler
 dataset:
   _component_: torchtune.datasets.alpaca_cleaned_dataset
-seed: null
+seed: 1372749820
```

Comment:
This might have been a wrong assumption on my part, I assumed that a seed was needed for testing purposes so the DataLoader would feed the same samples and we would see the same loss curve between runs, but perhaps this is not necessary